packages/tables/src/Concerns/CanSearchRecords.php

```diff
@@ -103,15 +103,23 @@ protected function applyColumnSearchesToTableQuery(Builder $query): Builder
         return $query;
     }
 
+    /**
+     * @return array<integer, string>
+     */
+    protected function searchWords(string $search): array
```

Comment:
Please name this `extractTableSearchWords()` and also use it within the "individual search" query for columns, so that also uses this behaviour

Reply:
Renamed the method and extended its use.

Not sure on the review - do I "resolve conversation" when I believe I've resolved, or do you do that when happy with the changes?